pkgs/development/tools/electron/binary/generic.nix

```diff
@@ -69,7 +70,7 @@ let
 
   headersFetcher =
     vers: hash:
-    fetchurl {
+    fetchzip {
       url = "https://artifacts.electronjs.org/headers/dist/v${vers}/node-v${vers}-headers.tar.gz";
```

Comment:
I recommend putting `name = "electron-{vers}-headers"` because the default name of  fetchzip based FOD (`"source"`) is misleading.